core/src/main/scala/org/http4s/headers/Last-Event-Id.scala

```diff
@@ -30,5 +30,9 @@ final case class `Last-Event-Id`(id: EventId) extends Header.Parsed {
 
 object `Last-Event-Id` extends HeaderKey.Internal[`Last-Event-Id`] with HeaderKey.Singleton {
   def parse(s: String): ParseResult[`Last-Event-Id`] =
-    HttpHeaderParser.LAST_EVENT_ID(s)
+    ParseResult.fromParser(parser, "Invalid Last-Event-Id header")(s)
+
+  private[http4s] val parser = (Parser.anyChar.rep0).string.map { (id: String) =>
```

Comment:
this is a one to one translation of the parboiled parser, but it doesn't seem to be particularly useful

Reply:
Per the [specification](https://html.spec.whatwg.org/multipage/server-sent-events.html#parsing-an-event-stream), it's anything _but_ a LF or CR.  Also interesting is that the data must be encoded as UTF-8, but it's not clear how this would be encoded into headers.  Smuggled in as obs-text, I guess?

Reply:
7a56bcdf6e2